File: pyfastlogging/examples/default.py
import fastlogging_rs as fl
from fastlogging_rs import (
    TRACE,
    Logging,
    ConsoleWriterConfig,
)

if __name__ == "__main__":
    fl.trace("Trace Message")
    fl.debug("Debug Message")
    fl.info("Info Message")
    fl.success("Success Message")
    fl.warning("Warning Message")
    fl.error("Error Message")
    fl.fatal("Fatal Message")

File: cxxfastlogging/examples/default.cpp
#include "cxxfastlogging/h/fastlogging.h"

int main()
{
    auto log = Logging::new_default();
    log->trace("Trace Message");
    log->debug("Debug Message");
    log->info("Info Message");
    log->success("Success Message");
    log->warning("Warning Message");
    log->error("Error Message");
    log->fatal("Fatal Message");
    log->shutdown(false);
    return 0;
}
